# Computing Codon Usage Bias in Propionibacterium acidifaciens

This is just a fun project where I calculated the highest codon frequency in Propionibacterium acidifaciens' complete genome and its four highly expressed genes (ribosomes and chaperones are usually highly expressed in cells, hence the judgment). 

Propionibacterium acidifaciens complete genome sequence: https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP033719.1?report=gbwithparts&log$=seqview

rsgA gene sequence (sequence.txt): https://www.ncbi.nlm.nih.gov/nuccore/NZ_LT593929.1?report=genbank&from=1396763&to=1397857

clpB gene sequence (clpB.txt): https://www.ncbi.nlm.nih.gov/nuccore/NZ_LT593929.1?report=genbank&from=481531&to=484131

rsfA gene sequence (rsfA.txt): https://www.ncbi.nlm.nih.gov/nuccore/NZ_LT593929.1?report=genbank&from=1573966&to=1574349&strand=true

dnaK gene sequence (dnaK.txt): https://www.ncbi.nlm.nih.gov/protein/AYW77536.1
